### PR TITLE
grid manual update and new docs for mesh reading

### PR DESCRIPTION
**Pull request by koehlerson**

### CODE REVIEW

File: docs/src/reference/grid.md [hunk 1]
```diff
@@ -56,6 +56,21 @@ addnodeset!
 create_coloring
 ```
 
+## Mesh Reading
+`FerriteGmsh.jl` supports all defined cells with an alias in (`Ferrite.jl`)(https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/Grid/grid.jl#L39-L54) as well as the 3D Serendipity `Cell{3,20,6}`.
```
Comment (fredrikekre) [inline docs/src/reference/grid.md:60]:
```suggestion
`FerriteGmsh.jl` supports all defined cells with an alias in [`Ferrite.jl`](https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/Grid/grid.jl#L39-L54) as well as the 3D Serendipity `Cell{3,20,6}`.
```

Comment (KnutAM) [reply docs/src/reference/grid.md]: Would you like me to contribute with an equivalent subsection about the `FerriteMeshParser.jl`'s interface?
Currently, only the MixedDofHandler example in #448 is using this.

Comment (koehlerson) [reply docs/src/reference/grid.md]: I would like it. Perhaps you could do a ### heading for FerriteGmsh and FerriteMeshParser?

**Comment by koehlerson** [comment]
should the added section be moved to manual?

**Comment by KnutAM** [comment]
> should the added section be moved to manual?

I think that would be good!
Then, perhaps it makes sense to rename the header to "How to obtain a mesh" (or grid),
and have sub-headers: `generate_grid`, `FerriteGmsh.jl`, `FerriteMeshParser.jl`?
I'm not sure if this should come before or after the description of the `Grid` type though. 
First-time users would primarily be interested in how to obtain meshes. But part of the descriptions rely on basic knowledge about the `Grid`-type

**Review by KnutAM — APPROVED** [review]
Looks good, just a few minor things that I saw. 
Perhaps we should change the PR name since this turned into restructuring the grid documentation as well.

File: docs/src/manual/grid.md [hunk 2]
```diff
@@ -3,10 +3,49 @@ DocTestSetup = :(using Ferrite)
...
-A Ferrite `Grid` can be generated with the `generate_grid` function. 
+
+## Mesh Reading
+
+A Ferrite `Grid` can be generated with the [`generate_grid`](@ref) function. 
 More advanced meshes can be imported with the 
 [`FerriteMeshParser.jl`](https://github.com/Ferrite-FEM/FerriteMeshParser.jl) (currently for Abaqus .inp files),
-or even created with the [`FerriteGmsh.jl`](https://github.com/Ferrite-FEM/FerriteGmsh.jl) package. 
+or even created and translated with the [`Gmsh.jl`](https://github.com/JuliaFEM/Gmsh.jl) and [`FerriteGmsh.jl`](https://github.com/Ferrite-FEM/FerriteGmsh.jl) package, respectively.
+For more information about this, consider the docs of [`FerriteGmsh.togrid`](@ref) as well as the [Stokes Flow](@ref) and [Incompressible Navier-Stokes Equations via DifferentialEquations.jl](@ref) example.
```
[the remark this answers is not in the document]
Comment (koehlerson) [reply docs/src/manual/grid.md:13]: yes

**Pull request merged**